share/spice/dogo_news/dogo_news.css

```diff
@@ -1,3 +1,18 @@
 .zci--dogo_news .tile--dogo_news .tile__foot {
     padding-bottom: 0.2em;
-}
\ No newline at end of file
+}
+
+#zci-dogo_news .tile__title
+{
```

Comment:
Let's keep with our existing CSS style -- please put the braces at the end of the previous line